src/generate.ts

```diff
@@ -21,35 +23,63 @@ const colors = [
     ],
   },
   {
-    skin: "#ddb7a0",
-    hair: [
-      "#272421",
-      "#3D2314",
-      "#5A3825",
-      "#CC9966",
-      "#2C1608",
-      "#e9c67b",
-      "#D7BF91",
-    ],
+    skin: ["#f2e4cb", "#f5dbad"],
+    hair: ["#272421", "#0f0902"],
   },
-  { skin: "#ce967d", hair: ["#272421", "#423125"] },
-  { skin: "#bb876f", hair: ["#272421"] },
-  { skin: "#aa816f", hair: ["#272421"] },
-  { skin: "#a67358", hair: ["#272421"] },
-  { skin: "#ad6453", hair: ["#272421"] },
-  { skin: "#74453d", hair: ["#272421"] },
-  { skin: "#5c3937", hair: ["#272421"] },
+  {
+    skin: ["#bb876f", "#aa816f", "#a67358"],
+    hair: ["#272421", "#1c1008"],
+  },
+  { skin: ["#ad6453", "#74453d", "#5c3937"], hair: ["#272421"] },
 ];
 
 const defaultTeamColors = ["#0d435e", "#f0494a", "#cccccc"];
 
 const roundTwoDecimals = (x: number) => Math.round(x * 100) / 100;
 
-const generate = (overrides: Overrides) => {
-  const eyeAngle = Math.round(Math.random() * 25 - 10);
+const generate = (overrides: Overrides, options: { chosenRace: Race }) => {
+  const playerRace: Race = (function () {
+    if (options == null) {
+      switch (Math.floor(Math.random() * 4)) {
+        case 0:
+          return "Caucasian";
+        case 1:
+          return "Asian";
+        case 2:
+          return "Latino";
+        case 3:
+          return "African";
+      }
+    }
+    return options.chosenRace;
+  })();
+  const eyeAngleAdjust = (() => {
```

Comment:
I think we should get rid of this. There are various facial features that vary across races, and in theory they could be supported here, but it probably should be done carefully to make sure it's done with appropriate sensitivity. (Not saying you did anything wrong, I'm just cautious.)

Reply:
Yea, I agree. Even I was unsure if I should add it, even though I tested it a little. I think in the future there should be some facial adjustment, but it should definitely be tested to make sure it will look normal, as random numbers can give us extreme examples.